.github/workflows/static-checks.yml

```diff
@@ -0,0 +1,55 @@
+name: Static check
+on: 
+  push:
+  workflow_dispatch:
+    inputs:
+        branch:
```

Comment:
I don't see that the input variable `branch` is used in this workflow
Should it be used?